buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -407,6 +407,7 @@ class BuildPlugin implements Plugin<Project> {
             systemProperty 'jna.nosys', 'true'
             // default test sysprop values
             systemProperty 'tests.ifNoTests', 'fail'
+            // TODO: remove setting logging level via system property
```

Comment:
I don't think we can do anything but through a system property. However, we should make it a test specific prop that is read by BootstrapForTesting (which should be initializing logging for tests...I was surprised to see there was no explicit init of logging there...)